src/Data/Functor/Interval.hs

```diff
@@ -52,11 +52,14 @@ module Data.Functor.Interval
 ) where
 
 import Control.Applicative (liftA2)
+import Control.DeepSeq (NFData)
 import Control.Monad.Trans.Class
 import Data.Coerce (coerce)
 import Data.Fixed (mod')
 import Data.Function (on)
+import Data.Functor.Classes (Show1, liftShowsPrec, Eq1, liftEq, Ord1, liftCompare)
```

Comment:
I don’t think we need to list the imports explicitly here, but if we’re going to, let’s sort this alphabetically. I think I have `stylish-haskell` config in this repo, if that helps.